libs/cardano-ledger-binary/src/Cardano/Ledger/Binary/Crypto.hs

```diff
@@ -85,14 +90,25 @@ decodeVerKeyKES :: C.KESAlgorithm v => Decoder s (C.VerKeyKES v)
 decodeVerKeyKES = fromPlainDecoder C.decodeVerKeyKES
 {-# INLINE decodeVerKeyKES #-}
 
-encodeSignKeyKES :: C.KESAlgorithm v => C.SignKeyKES v -> Encoding
-encodeSignKeyKES = fromPlainEncoding . C.encodeSignKeyKES
+encodeSignKeyKES :: (MonadST m, MonadThrow m, C.UnsoundKESAlgorithm v)
+                 => C.SignKeyKES v
+                 -> m Encoding
+encodeSignKeyKES = fmap fromPlainEncoding . C.encodeSignKeyKES
```

Comment:
This as well as instances for `Enc/DecCBOR` SignKeyKES can be removed from `cardano-ledger-binary`. Make sure to bump up the major version in the package and update CHANGELOG

We do not use KES in ledger, so there is no point for us providing them. It should be up to actual users of KES (i.e. consensus) to decide which methods o serialization they want to use